### ISSUE TITLE
Credo wrongly reports about macro name

### ISSUE THREAD

**Issue by luigi-discoup** [rrrene/credo#1100]
### Precheck

* [x] Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* [x] For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): `1.7.2-ref.main.312f3a4`
* Erlang/Elixir version (`elixir -v`): `Elixir 1.15.6 (compiled with Erlang/OTP 25)` | `Erlang/OTP 25 [erts-13.2.2.3] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1] [jit]`
* Operating system: `Debian GNU/Linux 11 (bullseye)`

### What were you trying to do?
I'm trying to define a macro predicate using the `is_something` naming convention. I've made a test with a sample code:

```elixir
test "it should NOT report a violation with defmacro" do
  """
  defmacro is_user(cookie) do
  end
  """
  |> to_source_file
  |> run_check(@described_check)
  |> refute_issues()
end
```

### Expected outcome
I believe  that no issue should be reported

### Actual outcome
Credo reports the following readability issue:

```
Predicate function names should not start with 'is', and should end in a question mark.
```

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by luigi-discoup** [comment]
Tested with version on `master` and now it works as expected. Waiting for the new release.

Thank you @rrrene!
